### DISCUSSION TITLE
installing on cpanel

### DISCUSSION THREAD

**Discussion by mecpcanada** [userjack6880/Open-DMARC-Analyzer#100] [Q&A]
Hi any way to install this on a cpanel ? is there a guide for it i tried to do it i got this error 
connecting to database... success
opening file... success
→ CREATE VIEW IF NOT EXISTS report_stats AS (
→ SELECT
→ report.serial, domain, rcount, disposition, reason,
→ policy_p, policy_pct, dkimdomain, dkimresult, dkim_align,
→ spfdomain, spfresult, spf_align, mindate, maxdate
→ FROM report RIGHT JOIN rptrecord
→ ON report.serial=rptrecord.serial
→ );
performing query... failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF NOT EXISTS report_stats AS ( SELECT report.serial, domain, rcount, disposi' at line 1

**Comment by userjack6880** [comment]
Please submit this to https://github.com/userjack6880/Open-DMARC-Analyzer/issues

The template should guide you to providing all the information needed to help you.
